packages/react-examples/package.json

```diff
@@ -53,7 +53,7 @@
     "@fluentui/react-hooks": "^8.2.2",
     "@fluentui/react-icons-mdl2": "^1.1.3",
     "@fluentui/react-label": "^9.0.0-alpha.9",
-    "@fluentui/react-link": "^9.0.0-alpha.48",
+    "@fluentui/react-link": "^9.0.0-alpha.47",
```

Comment:
react-link was migrated to new DX thus it was removed from these dependencies - probably bad rebase?